region/brinstar/green/Early Supers Room.json

```diff
@@ -978,6 +1030,43 @@
         "After the crumbles are broken, shoot the ceiling block and wait by the door for the Waver to come."
       ]
     },
+    {
+      "link": [2, 4],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        {"or": [
+          "canWalljump",
+          "HiJump",
+          "SpaceJump",
+          "canIBJ",
+          "canSpringBallJumpMidAir",
+          "h_crouchJumpDownGrab"
+        ]},
+        {"or": [
+          "h_CrystalFlash",
+          {"and": [
+            {"disableEquipment": "ETank"},
+            {"resourceAvailable": [{"type": "Energy", "count": 80}]},
+            {"resourceMissingAtMost": [{"type": "Missile", "count": 0}]},
+            {"resourceMissingAtMost": [{"type": "Super", "count": 0}]},
```

Comment:
There's enough Wavers up there that it's not exactly necessary to rely on being full Missiles/Supers. If we put the assumed energy available to 95, that could work because it would be almost guaranteed to get at least one energy drop. On the other hand, Missiles and Supers can also be farmed here, so being full on them is also not much of a restriction. If the neighboring room involved a heated R-mode setup, the energy requirement could possibly be the more impactful one. So probably it is best done the way you have it here.